### PR TITLE
Expose logits in FooProbs distributions

### PR DESCRIPTION
**Pull request by fehiepsi**
As title. This is to be closer to Pyro API and makes things easier to debug.

### CODE REVIEW

**Review by fritzo — APPROVED** [review]

**Comment by fehiepsi** [comment]
Thanks for reviewing!
